src/engine/client/client.h

```diff
@@ -36,13 +36,15 @@ class CSmoothTime
 	CGraph m_Graph;
 
 	int m_SpikeCounter;
+	int m_BadnessScore; // ranges between -100 (perfect) and MAX_INT
```

Comment:
Is it possible that the number overflows?
There's no clamping in the code.

Reply:
Should be impossible. Even if it does happen, it will quickly converge to -100, and it will only result in 5/5 being wrongly displayed for a while.